### ISSUE TITLE
Problem when install Oh My Bash with git plugin. 

### ISSUE THREAD

**Issue by jogampo** [ohmybash/oh-my-bash#306]
when I install oh My bash I see the next message:

bash: /home/<user>/.oh-my-bash/plugins/git/git.plugin.sh: línea 107: error sintáctico cerca del elemento inesperado `('
bash: /home/<user>/.oh-my-bash/plugins/git/git.plugin.sh: línea 107: `ggf() {'

**Comment by akinomyoga** [comment]
Thank you for the report. It doesn't seem to reproduce in my environment, but I guess an alias `ggf` is defined before sourcing `oh-my-bash.sh`.  At least, I get similar error messages when I define an alias before sourcing `oh-my-bash.sh`.

To check whether the alias `ggf` is defined or not, could you temporarily insert the following line in your `.bashrc` and check the output on the startup of Bash? Thank you.

```diff
diff --git a/.bashrc b/.bashrc
index ad745cd..346242d 100644
--- a/.bashrc
+++ b/.bashrc
@@ -83,6 +83,7 @@ plugins=(
 #      plugins+=(tmux-autoattach)
 #  fi

+alias ggf
 source "$OSH"/oh-my-bash.sh

 # User configuration
```

**Comment by jogampo** [comment]
Exactly, I have this alias into .Bash_aliases file and I didn't remember. I rename this file and reinstall Oh My Bash and all ok. 

Thanks for you response.

I close the issue.
